manuskript/exporter/manuskript/HTML.py

```diff
@@ -15,8 +15,8 @@
 
 class HTML(markdown):
     name = "HTML"
-    description = qApp.translate("Export", "Basic HTML output using python module 'markdown'.")
```

Comment:
Another minor suggestion.  I think the word "the" should be in lowercase.  For example:  "Basic HTML output using **t**he Python module 'markdown'."